docs/faq/what-is-a-build.md

```diff
@@ -28,5 +28,5 @@ If you look at the [other tutorials](https://docs.nextstrain.org/en/latest/tutor
 While it is possible to run a build by running each of the individual steps, we typically group these together into a make-type file.
 [Snakemake](https://snakemake.readthedocs.io/en/stable/index.html) is "a tool to create reproducible and scalable data analyses... via a human-readable, Python-based language."
 
-> Snakemake is installed as part of the [conda environment](https://docs.nextstrain.org/en/latest/guides/install/local-installation.html#install-augur-auspice-with-conda) or the [docker container](https://docs.nextstrain.org/en/latest/guides/install/cli-install.html).
+> Snakemake is installed as part of the [conda environment](https://docs.nextstrain.org/en/latest/guides/install/local-installation.html) or the [docker container](https://docs.nextstrain.org/en/latest/guides/install/cli-install.html).
```

Comment:
_(non-blocking)_

The link doesn't make sense for the link text "conda environment" but I think the only way to address this is to update this entire sentence with newer terminology such as "runtime". That is out of scope of this PR. This change seems appropriate for the sake of satisfying `linkcheck`.

Reply:
Created #1582 to track